modules/template/template.go

```diff
@@ -104,6 +105,15 @@ func NewFuncMap() []template.FuncMap {
 		"ThemeColorMetaTag": func() string {
 			return setting.ThemeColorMetaTag
 		},
+		"TabSizeClass": func(ec *editorconfig.Editorconfig, filename string) string {
+			if ec != nil {
+				def := ec.GetDefinitionForFilename(filename)
+				if def.TabWidth > 0 {
+					return fmt.Sprintf("tab-size-%d", def.TabWidth)
+				}
+			}
+			return "tab-size-8"
```

Comment:
+1 for 4 spaces by default. I really think that's common as the main discussion is always "4 spaces vs tabs". If anyone has any stats it would be very useful, but otherwise I would subjectively use 4 spaces by default.